### ISSUE TITLE
Problems with new build version and current instructions

### ISSUE THREAD

**Issue by c-born** [WAGO/tp-firmware-sdk#7]
I've been happily building with the Linaro 2017 toolchain for some time, and thought I should see if the updated version (FW19?) had any advantages.
I started with a clean build (new VM), but soon ran into some problems when following the instructions.
At section
`4.1) Select "software config" we want to build`
I get:

`:~/wago/ptxproj$ ptxdist select configs/vtp-ctp/ptxconfig_generic`
`ptxdist: error: ptxdist-2017.11.1 not found!`

Is this because `~/wago/ptxproj/configs/vtp-ctp/ptxconfig_generic` contains near the top
`# PTXdist 2017.11.1`
or has something else not been updated?

More confusing because at section:
`3.5) Install the build environment tool "ptxdist"`
it says
`Default setup installs build environment to folder: /usr/local/lib/ptxdist-2017.11.1/bin/ptxdist`

while it actually appears to install to
`/usr/local/lib/ptxdist-2020.08.0`

Are there compelling reasons to update to FW19 or FW20?
I'm happy with the 2017 build on my development system, but I have 22 brand new production units waiting for me to install our custom builds on, and I'd hate them to go out with security or other issues.

**Comment by c-born** [comment]
After hearing nothing back on this for a couple of months, I decided to have another go at getting a working build of a version that matches that in the 22 machines I have waiting to go out into production. This is what worked for me, it may be of help to others.

1. At step 1.3, after you clone https://github.com/WAGO/tp-firmware-sdk.git into ~/wago/ptxproj, you need to check-out tag V03.06.11-FW18.1 as the current head doesn't build due to old config files
2. At step 3.2 after you clone http://github.com/wago/ptxdist.git into ~/ptxdist, check-out commit WAGO/ptxdist@8d495aa
3. At step 4.5 when you run `ptxdist menuconfig`, before exiting track down and remove the CodeSys3 stuff as that causes the build to fail. (Unless you use it - then you are stuck until @Thomas-Brandt fixes it).
4. If you are planning to use VisualGDB then while you are in menuconfig you may want to to to busybox options and allow `stat` to accept the `-c` option which it appears to require.

Obviously this is a bit of a hacky work-around, and hopefully @Thomas-Brandt  or someone from Wago will have a chance to get the HEAD working or amend the instructions soon.

**Comment by Thomas-Brandt** [comment]
Hi.
If you are missing ptxdist 2017.11.1 please take a look at
https://github.com/WAGO/ptxdist/releases/tag/Ptxdist-2017.11.1
the problem is that PFC is already updated to ptxdist 2020.08.0 but not the TP.
This will be fixed with the upcoming SDK.

Otherwise there I don't see any other problems. 
Codesys3 has no build problems on our build-servers and on my machine.
If it is failing on your machine please provide a log that I could take a look at and help you with the problem.

**Comment by c-born** [comment]
@Thomas-Brandt Thanks for making those changes. I've just done a successful complete fresh default build using` ptxdist-2017.11.1` and current head. It only stopped twice, once to ask for the `texlive `package to be installed, the other was a fail that went away after I restarted the build using -j1.
Build took 2 days, most of it building the qt package.
I made some minor changes which had nothing to do with qt, (adding -c option to busybox stat), and the rebuild has been on qt for the last 12 hours - why? 
 
Instructions say to configure using:

`/usr/sbin/cbm`

I can't find this on the installed image (anywhere), although I can see it in the project files on the build host, which is weird!
